linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/ServerConfiguration.scala

```diff
@@ -207,4 +207,7 @@ object ServerConfiguration extends Logging {
   val LINKIS_SERVER_SESSION_PROXY_TICKETID_KEY =
     CommonVars("wds.linkis.session.proxy.user.ticket.key", "linkis_user_session_proxy_ticket_id_v1")
 
+  val LINKIS_SERVER_ENTRANCE_HEADER_KEY =
+    CommonVars("wds.linkis.server.entrance.header.key", "jobInstanceKey")
```

Comment:
rename "wds.linkis.server.entrance.header.key" to "linkis.server.entrance.header.key"

Reply:
done